sandbox-hpc.tfvars

```diff
@@ -0,0 +1,21 @@
+is_staging                                 = false
+is_production                              = false
+terraform_remote_state_dynamodb_table      = "terraform-state-lock-table-sandbox-hpc"
+terraform_remote_state_bucket_name         = "obi-tfstate-sandbox-hpc"
+nise_dockerhub_password                    = "placeholder"
+cell_svc_bucket_name                       = "sbo-cell-svc-perf-test-sandbox-hpc"
+ml_paper_bucket_name                       = "ml-paper-bucket-sandbox-hpc-test"
+nexus_domain_name                          = "sandbox-hpc.openbluebrain.com"
+nexus_obp_bucket_name                      = "nexus-obp-sandbox-hpc-test"
+nexus_ship_bucket_name                     = "nexus-ship-sandbox-hpc-test"
+nexus_openscience_bucket_name              = "nexus-openscience-sandbox-hpc-test"
+ec_apikey                                  = "placeholder"
+coreservices_public_key                    = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCy8UW1JdUjIuiiaI+xFId3smXFe6iwxdn6Klqi8y4E+cFHP/uQxW4AuCfJVoOjOQ2CNU0UIGRlCkw3IwUmZvwGjFZS1Vs6jt+KjWiDRdFzQQrz5+vuqPX2576wXlT+EHe0W6r7Qla5i1L6cjz6/E4u5yFZ3MJQSBYjLqXjT7Da4R72gPx9oiIoSH2JBu3vHyfkTEo3l6C+WJlYnUGOLnUxGGGnhJrBKmIRNMmtRNgQlBkPR4mnCAFABJfgpzgNf4bpqLbma5DabQGbvpX6qCLSAV5Zdd4gBQLIAJfS/a5pMWaIy9qDTWB2vy4Z39HI45k6efrLL+xgo+XYSRqn5jDT heeren@Mac"
+nexus_az_letter_id                         = "a"
+core_web_app_docker_image_url              = "bluebrain/sbo-core-web-app:2025.1.0-prod"
+virtual_lab_manager_docker_image_url       = "bluebrain/obp-virtual-lab-api:2025.1.0-prod"
+hpc_resource_provisioner_container_version = "0.5.10.dev36"
```

Comment:
CI cannot run because:

`bluenaas_docker_image_url                  = "bluebrain/blue-naas-single-cell:staging"`

missing

Reply:
Fixed with the latest rebase :-)